css/styles.css

```diff
@@ -155,168 +162,174 @@ html {
 
 /* card */
 
-
-
 /* product listing page */
 
-.ecomm-master-container{
-
-background-color: aqua;
-
-display: flex;
-justify-content: flex-end;
-}
-
-
-
-
-
-.sidebar{
-
-background-color: blue;
+.ecomm-master-container {
+  display: flex;
+  justify-content: flex-end;
 }
 
-.ecomm-cards{
-  width: calc(100% - 25rem);
-background-color: red;
+.sidebar {
+  top: 6rem;
 
-}
 
-/* product listing page */
 
+display: flex;
 
-/* input range slider */
-.k-input-range-mastercontainer{
-
-background-color: aquamarine;
+flex-direction: column;
 
+gap: 15px;
+padding: 1rem;
 
 }
 
-.k-range{
-
+.sidebar-option-center{
 
 display: flex;
-flex-direction: column;
-justify-content: center;
-align-items: center;
-}
+gap: 5px;
 
+}
 
-.k-range-content{
+.k-radio-container{
 
-display: flex;
+  display: flex;
+  gap: 5px;
+  
 
-width: 20rem;
-justify-content: space-between;
 
 }
 
-.k-range-input-holder{
-
-
-display: flex;
 
+.ecomm-cards {
+  width: calc(100% - 25rem);
+}
 
-justify-content: center;
+/* product listing page */
 
+/* input range slider */
+.k-input-range-mastercontainer {
 }
 
+.k-range {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
 
-.k-range-input-holder input{
-
+.k-range-content {
+  display: flex;
 
-width: 20rem;
+  width: 20rem;
+  justify-content: space-between;
+}
 
+.k-range-input-holder {
+  display: flex;
 
+  justify-content: center;
 }
 
+.k-range-input-holder input {
+  width: 20rem;
+}
 
 /* input range slider */
 
-
-
-
-
 /* ecommm cards container */
 
-.ecomm-heading{
-
-padding: 1rem;
-display: flex;
-  
- gap: 10px;
-align-items: baseline;
-
-flex-direction: row;
+.ecomm-heading {
+  padding: 1rem;
+  display: flex;
 
+  gap: 10px;
+  align-items: baseline;
 
+  flex-direction: row;
 }
 
-
 /* ecommm cards container */
 
-
-
 /* k-card */
 /* card-basic flex-div flex-div-direction-col background-primary box-shadow border-radius-all-8 color-primary-color-dark */
 
-
-.yo{
-
-background-color: #2d3748;
-
-
+.yo {
+  background-color: #2d3748;
 }
-.k-card{
-display: flex;
+.k-card {
+  display: flex;
 
-width: 350px;
+  width: 350px;
```

Comment:
might want to use rem for responsiveness

Reply:
i kept it in px so that i could have a constant width irrespective of screen size, perhaps media query is the way..